dotnet/InkRecognition/wpf-app/README.md

```diff
@@ -42,17 +35,35 @@ This project framework provides the following features:
 ### Prerequisites
 
 * [Visual Studio](https://docs.microsoft.com/en-us/visualstudio/install/install-visual-studio)
-* Windows SDK Version >= 10.0.18362. You can install it from Visual Studio Installer under **Indivial components** tab, or download from [Windows Insider Preview SDK](https://www.microsoft.com/en-us/software-download/windowsinsiderpreviewSDK)
-  * If you install the Windows SDK into a customized path, please edit **src\NoteTaker.csproj** to change the HintPath of **Windows.winmd** to the correct path 
-* Requires subscription key from [Azure Cognitive Services](https://docs.microsoft.com/en-us/azure/cognitive-services/authentication) 
-
-## Build the Sample
-
-1. Clone or download this sample repository `git clone https://github.com/Azure-Samples/cognitive-services-csharp-wpf-ink-recognition.git` 
-2. Start Visual Studio and select **File > Open > Project/Solution** and open "NoteTaker.sln" 
-3. Replace "[YOUR SUBSCRIPTION KEY]" in MainWindows.xaml.cs with valid subscription key
-4. Press Ctrl+Shift+B, or select **Build > Build Solution**
-
-## Run the sample
-
-To debug the sample and then run it, press F5 or select Debug > Start Debugging. To run the sample without debugging, press Ctrl+F5 or select Debug > Start Without Debugging.
\ No newline at end of file
+* Windows SDK Version >= 10.0.18362. You can install it from Visual Studio Installer under **Indivial components** tab, or download from [Windows 10 SDK](https://developer.microsoft.com/en-us/windows/downloads/windows-10-sdk)
+  * If you install the Windows SDK into a customized path, please edit **src\NoteTaker.csproj** to change the HintPath of **Windows.winmd** to the correct path
+* Nuget Packages
+  * [Newtonsoft.JSON](https://www.newtonsoft.com/json)
+  * [Microsoft.Net.Http](https://www.nuget.org/packages/Microsoft.Net.Http/)
+  * [Xaml Island for WPF](https://www.nuget.org/packages/Microsoft.Toolkit.Wpf.UI.Controls)
+* Subscription key from [Azure Cognitive Services](https://docs.microsoft.com/en-us/azure/cognitive-services/authentication)
+* Windows Build 1903 or higher is required for Xaml island features
+
+### Build the Sample
+
+1. Clone or download this sample repository `git clone https://github.com/Azure-Samples/cognitive-services-csharp-wpf-ink-recognition.git`
+2. Start Visual Studio and select **File > Open > Project/Solution** and open **src\NoteTaker.sln**
+3. Replace "[YOUR SUBSCRIPTION KEY]" in **src\MainWindows.xaml.cs** with valid subscription key
+4. Select **Build > Build Solution**
+
+### Run the sample
+
+* To debug the sample and then run it, select **Debug > Start Debugging**. To run the sample without debugging, select **Debug > Start Without Debugging**.
+* Write down something one the upper rectangle of the app
```

Comment:
on

Reply:
"rectangle" might be confusing! Please repharse the instruction. Same applies to L58

Reply:
Is "region" better?